### DISCUSSION TITLE
Can I set a CD offset with Freac?

### DISCUSSION THREAD

**Discussion by Wozza2014** [enzo1982/freac#476] [Help & Support]
Hi.
 
I've EAC for years to rip music CD's and with that program you need to set the correct CD offset for different CD drives.  Is this possible with Freac?

Thanks.

**Comment by enzo1982** [comment]
Yes, you can configure the drive's read offset in the Ripper settings in fre:ac's configuration dialog:

![grafik](https://user-images.githubusercontent.com/7796713/232617556-c3fd07f6-7629-480b-ae1f-c8fb8fae1cba.png)

fre:ac will also attempt to automatically configure the read offset using the AccurateRip database when no offset was configured before ripping the first CD.
